www/book.css

```diff
@@ -48,10 +54,10 @@ a:active { color: #f64; }
 
 li { margin: 1em 0; }
 
-.main ol, .main > h1 { font-size: 120%; }
+.main ol, .main h1 { font-size: 120%; }
```

Comment:
Why is this change necessary?

Reply:
Ah—sort of bizarrely the "Part XXX" headers were bigger than the "Introduction" and "Conclusion" headers because the latter are inside a <section>.

Reply:
Can you add a comment explaining?